### PR TITLE
Create application process file .yaml

### PR DESCRIPTION
**Pull request by blackinno**
- remove the application folder.
- create `application.yaml` as process file.
- update README.md
- update the environment example file.
- update docker-compose.yaml on the website folder.

### CODE REVIEW

File: application.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  - type: task
+    key: calculate-amount
+    instance:
+      src: https://github.com/mesg-foundation/service-js-function
+    taskKey: execute
+    inputs:
+      code: 'module.export = (value) => value * 0.4 * 100'
+      inputs:
+        stepKey: request-data
+        key: result.number
```
Comment (antho1404) [inline application.yaml]: Maybe these 2 tasks could be merged into one with a function like 
```javascript
module.export = data => ({
  email: data.email,
  number: data.number,
  ethAddress: data.ethAddress,
  credit_card_token: data.token,
  amount: data.number * 0.4 * 100
})
```

File: application.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        key: result.credit_card_token
+  - type: task
+    key: erc20-transfer
+    instance:
+      src: https://github.com/mesg-foundation/service-ethereum-erc20
+      env:
+        - PROVIDER_ENDPOINT=$(env:PROVIDER_ENDPOINT)
+    taskKey: transfer
+    inputs:
+      privateKey: "0x$(env:PRIVATE_KEY)"
```
Comment (antho1404) [inline application.yaml]: I think we should just read `PRIVATE_KEY` and make sure in the readme or the `.env.example` that it is prefixed by `0x`

File: .env.example [hunk 3]
```diff
@@ -1,3 +1,4 @@
 STRIPE_SECRET=...
 PRIVATE_KEY=...
```
Comment (antho1404) [inline .env.example]:
```suggestion
PRIVATE_KEY=0x...
```

File: .env.example [hunk 4]
```diff
@@ -1,3 +1,4 @@
...
-SENDGRID_API_KEY=...
\ No newline at end of file
+SENDGRID_API_KEY=...
```
Comment (antho1404) [inline .env.example]: Let's prefix with the prefix of sendgrid `SG.` if I remember but that's to confirm
```suggestion
SENDGRID_API_KEY=SG....
```

File: .env.example [hunk 5]
```diff
@@ -1,3 +1,4 @@
 STRIPE_SECRET=...
```
Comment (antho1404) [inline .env.example]: Also to prefix to give an indication. I think it's something like `sk_test` or `sk_live`

```suggestion
STRIPE_SECRET=sk_test...
```

**Review by antho1404 — COMMENTED** [review]
The readme needs some update, the `start` script doesn't exist anymore.
Also, we could remove the docker-compose and just add in the readme the following to start the website.
```
docker build website -t erc20-stripe-website
docker run -d -p 8080:80 erc20-stripe-website
```

**Comment by blackinno** [comment]
Fixed, it's done.

**Review by antho1404 — APPROVED** [review]

**Pull request merged**